src/helpers/xml-builder.js

```diff
@@ -1315,6 +1383,42 @@ const computeImageDimensions = (vNode, attributes) => {
   let modifiedMaxHeight;
   let modifiedMaxWidth;
 
+  // Check for HTML width and height attributes first (e.g., from TinyMCE)
+  if (vNode?.properties?.attributes) {
+    const htmlWidth = vNode.properties.attributes.width;
+    const htmlHeight = vNode.properties.attributes.height;
+
+    if (htmlWidth) {
+      // HTML attributes without units default to pixels
+      const hasUnits =
+        pixelRegex.test(htmlWidth) ||
+        percentageRegex.test(htmlWidth) ||
+        pointRegex.test(htmlWidth) ||
+        cmRegex.test(htmlWidth) ||
+        inchRegex.test(htmlWidth);
+      const widthWithUnits = hasUnits ? htmlWidth : `${htmlWidth}px`;
+      modifiedWidth = calculateAbsoluteValues(widthWithUnits, originalWidthInEMU);
+    }
+    if (htmlHeight) {
+      // HTML attributes without units default to pixels
+      const hasUnits =
+        pixelRegex.test(htmlHeight) ||
+        percentageRegex.test(htmlHeight) ||
+        pointRegex.test(htmlHeight) ||
+        cmRegex.test(htmlHeight) ||
+        inchRegex.test(htmlHeight);
+      const heightWithUnits = hasUnits ? htmlHeight : `${htmlHeight}px`;
```

Comment:
should be refactored into a common function, that iterates over various regexes and calculates the values